Xamarin.Essentials/Launcher/Launcher.netstandard.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
```

Comment:
This PR is about fixing Clipboard support for GTK, the Launcher stuff should go in another PR (or at least in a different commit)